hazelcast/src/main/java/com/hazelcast/internal/util/OperatingSystemMXBeanSupport.java

```diff
@@ -27,12 +27,24 @@
  */
 public final class OperatingSystemMXBeanSupport {
 
+    static final String COM_HAZELCAST_FREE_PHYSICAL_MEMORY_SIZE_DISABLED = "com.hazelcast.FreePhysicalMemorySizeDisabled";
```

Comment:
Minor: to be consistent with the other flags, consider changing it to something like `hazelcast.os.free.physical.memory.disabled`

Reply:
Done